Getting cannot find module error

```
$ npm i -g travis-deploy-once@5.0.0
/home/travis/.nvm/versions/node/v8.11.4/bin/travis-deploy-once -> /home/travis/.nvm/versions/node/v8.11.4/lib/node_modules/travis-deploy-once/bin/travis-deploy-once.js
+ travis-deploy-once@5.0.0
added 252 packages in 4.749s
after_success.2
$ travis-deploy-once "npm run semantic-release"
module.js:549
    throw err;
    ^
Error: Cannot find module '../babel-register'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/home/travis/build/satyarohith/shark/node_modules/travis-deploy-once/bin/travis-deploy-once.js:4:1)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
```
I used latest version, tried v5 and v4.4.1 getting the same error. I coudn't figureout what I was doing wrong and also I checked `node_modules/travis-deploy-once` for `babel-register.js` and the file was not found.
Thanks for taking time!

--- a/package.json
+++ b/package.json
@@ -52,6 +52,7 @@
   "files": [
     "bin",
     "lib",
+    "babel-register",
     "index.js",
     "cli.js"
   ],

